aiidalab_widgets_base/viewers.py

```diff
@@ -348,8 +348,21 @@ def _observe_cell(self, _=None):
                 spglib_structure, symprec=1e-5, angle_tolerance=1.0
             )
 
+            periodicity_map = {
+                (True, True, True): "xyz",
+                (True, False, False): "x",
+                (False, True, False): "y",
+                (False, False, True): "z",
+                (True, True, False): "xy",
+                (True, False, True): "xz",
+                (False, True, True): "yz",
+                (False, False, False): "-",
+            }
             self.cell_spacegroup.value = f"Spacegroup: {symmetry_dataset['international']} (No.{symmetry_dataset['number']})"
             self.cell_hall.value = f"Hall: {symmetry_dataset['hall']} (No.{symmetry_dataset['hall_number']})"
+            self.periodicity.value = (
+                f"Periodicity: {periodicity_map[tuple(self.structure.pbc)]}"
+            )
         else:
```

Comment:
Could you please reset the following widgets here as well:

```python
            self.cell_spacegroup.value = ""
            self.cell_hall.value = ""
            self.periodicity.value = ""
```

Otherwise, if no structure is selected, the values of the previous structure are still shown.